src/main/java/com/hubspot/jinjava/lib/filter/TrimFilter.java

```diff
@@ -28,7 +26,9 @@ public String getName() {
 
   @Override
   public Object filter(Object var, JinjavaInterpreter interpreter, String... args) {
-    return StringUtils.trim(Objects.toString(var));
+    if (var instanceof String) {
+      return safeFilter(StringUtils.trim(var.toString()), interpreter, args);
```

Comment:
Should this be `return StringUtils.trim(Objects.toString(var));` instead?

Reply:
Why? If we know its a string, whats the need to wrap it in the call. We know we can cast or do .toString